src/azure-cli/setup.py

```diff
@@ -154,9 +153,6 @@
 with open('HISTORY.rst', 'r', encoding='utf-8') as f:
     HISTORY = f.read()
 
-if sys.version_info < (3, 4):
```

Comment:
Why just remove this statement instead of changing it to `if sys.version_info < (3, 6)`?

Reply:
Same question.

Reply:
This check was added in this [PR](https://github.com/Azure/azure-cli/pull/11363). Adding this sets the hard limit for python versions at installation and block lower verisons. When we remove a version from the classifier, we're droppiong support for that version. It doesn't mean the code must not be compatible with that version. We can remove python 3.6 simply becaused it reaches end-of-life in the future, but our CLI code could still be totally compatible with python 3.6 at that time until at one point it's not. We will not fix it as we stated it's not supported.

Reply:
@fengzhou-msft I think I got your point, thanks for the clarification.